pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -69,7 +69,7 @@ one = 'Describe, list, and delete consumer groups for the current Apache Kafka i
 
 [kafka.consumerGroup.cmd.longDescription]
 one = '''
-Use these commands to describe, list, and delete consumer groups for the current Kafka instance.'
+Use these commands to describe, list, and delete consumer groups for the current Kafka instance.
```

Comment:
Looking at the other _one_ statements, does this actually need a quote on either end?
Or aren't they needed?
Referring to 'Reset consumer group offsets on a specified topic' below